tests/check_new_syntax.py

```diff
@@ -80,10 +84,12 @@ class TextFinder(ast.NodeVisitor):
         def visit_ImportFrom(self, node: ast.ImportFrom) -> None:
             if node.module == "typing" and any(thing.name == "Text" for thing in node.names):
                 errors.append(f"{path}:{node.lineno}: Use `str` instead of `typing.Text` in a Python-3-only stub.")
+            self.generic_visit(node)
 
         def visit_Attribute(self, node: ast.Attribute) -> None:
             if isinstance(node.value, ast.Name) and node.value.id == "typing" and node.attr == "Text":
```

Comment:
We should put this in flake8-pyi instead.

Reply:
It's on my to-do list :)